challenges-36-plus.yml

```diff
@@ -0,0 +1,53 @@
+modal_redirect:
+  question: |
+    Which of these properties do *not* come in the `fetchResponse` object
+    when the event `turbo:before-fetch-response` happens?
+  explanation: |
+    Feel free to explore the `fetchResponse` object in your browser console.
+    It contains many useful properties for you to make your Turbo applications
+    really powerful!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          `response`: Is an object that contains details about the server
+          response, such as `body`, `headers` `status`, `redirected`
+          and more.
+        b: |
+          `location`: Is an object that contains the information about the
+          location where the response came from.
+        c: |
+          `language`: Is a string containing the language in which our content
+          was retrieved.
+        d: |
+          `succeded`: Is a boolean that determines weather the fetch was
```

Comment:
Is this the right key? It looks like a typo - maybe `succeeded` - so we should check

Reply:
I do have a typo there! I'll doublecheck